### DISCUSSION TITLE
Mobilenet SSD equivalent

### DISCUSSION THREAD

**Discussion by paullee2018** [larq/compute-engine#625] [General]
Is there a binarized mobilenet SSD equivalent? Does it support residual/skip connection?

**Comment by timdebruin** [comment]
Hi Paul, thank you for your interest in Larq! 

MobileNets have been found to be hard to quantize below 8 bits. There is work on it (see for example Phan et. al., ["Binarizing MobileNet via Evolution-based Searching"](https://arxiv.org/pdf/2005.06305.pdf), arXiv, 2020 ). However, for object detection with a BNN backbone, it might be easier to start with a network that was designed as a BNN in the first place. [Here](https://github.com/tehtea/QuickYOLO) is an example from the community of a repository that uses QuickNet as a backbone.  

Residual/skip connections are definitely supported in larq/LCE - they are in fact quite important for getting BNNs to work. 

I hope this helps, feel free to ask follow up questions!

**Reply by paullee2018** [reply]
I am currently looking at this repo.

https://github.com/ZiweiWangTHU/BiDet

I might try out both solution to check which one is suited for my application.

Thanks.
